amqp/types.py

```diff
@@ -0,0 +1,566 @@
+"""Abstract types."""
+import abc
+import asyncio
```

Comment:
should we just use asyncio for draft now or switch to anyio/trio?

Reply:
We should use anyio for generic clients.
This library is useful as a standalone as well.

Reply:
btw, except https://github.com/agronholm/anyio/tree/master/docs docs I couldn't find another docsite.  thought you might have known?